geotrek/api/v2/views/tourism.py

```diff
@@ -84,8 +84,9 @@ class TouristicEventViewSet(api_viewsets.GeotrekGeometricViewset):
     filter_backends = api_viewsets.GeotrekGeometricViewset.filter_backends + (
         api_filters.GeotrekTouristicEventFilter,
         api_filters.NearbyContentFilter,
-        api_filters.UpdateOrCreateDateFilter
+        api_filters.UpdateOrCreateDateFilter,
     )
+    filterset_fields = ["bookable"]  # TODO find a way to overide automatic Swagger doc to add a description for this filter
```

Comment:
Many ways to add doc : 
  - default is model field help_text
  - specific for API : use filterset_class instead of filterset_fields, inheritad from djang_filters.FilterSet. and customize bookable field with specific help_text